inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/RdfUtils.java

```diff
@@ -77,12 +90,27 @@
             throws QueryEvaluationException {
         return conn.getStatements(subj, pred, obj, includeInferred);
     }
-    
+
     public static RepositoryResult<Statement> getStatementsSparql(RepositoryConnection conn,
-            Resource subj, IRI pred, Value obj, boolean includeInferred)
+        Resource subj, IRI pred, Value obj, boolean includeInferred)
+        throws QueryEvaluationException
+    {
+        return getStatementsSparql(conn, subj, pred, obj, includeInferred, null);
+    }
+
+    public static RepositoryResult<Statement> getStatementsSparql(RepositoryConnection conn,
+            Resource subj, IRI pred, Value obj, boolean includeInferred, String language)
         throws QueryEvaluationException
     {
-        String QUERY = "SELECT * WHERE { ?s ?p ?o } LIMIT 1000";
+        String filter = "";
+        if (language != null) {
+            filter = "FILTER(LANG(?o) = \"\" || LANGMATCHES(LANG(?o), \"" + language + "\")).";
+        }
+        String QUERY = String.join("\n",
+            "SELECT * WHERE { ",
+            "?s ?p ?o ",
+            filter,
+            "} LIMIT 1000");
```

Comment:
Do we have a constant for `LIMIT` somewhere?

Reply:
No.

Reply:
Done. https://github.com/inception-project/inception/issues/300